index.html

```diff
@@ -0,0 +1,93 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Delicious</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/layout.css">
+    <link rel="stylesheet" href="css/style.css">
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7/jquery.js"></script>
+    <script src="effects.js"></script>
+  </head>
+  <body>
+    <div class="body">
+      <div class="heading">
+        <header>
+          <img class="heading__logo" src="images/logo.png" alt="Delicious Logo"/>
+          <h1 class="heading__title">Delicious</h1>
+          <h2 class="heading__subtitle">The Best Food Blog On The Web.</h2>
+        </header>
+        <nav class="heading__nav">
+          <ul>
+            <li class="heading__nav__icon"><img src="images/fb-icon.png" alt="Facebook Icon"></li>
+            <li class="heading__nav__icon"><img src="images/twit-icon.png" alt="Twitter Icon"></li>
+            <li class="heading__nav__icon"><img src="images/gp-icon.png" alt="Google Plus Icon"></li>
+            <li class="heading__nav__icon"><img src="images/insta-icon.png" alt="Instagram Icon"></li>
+            <li class="heading__nav__icon"><img src="images/flic-icon.png" alt="Flic Icon"></li>
+            <li class="heading__nav__icon"><img src="images/pint-icon.png" alt="Pinterest Icon"></li>
+            <li class="heading__nav__icon"><img src="images/rss-icon.png" alt="RSS Icon"></li>
+            <li class="heading__nav__icon"><img src="images/mail-icon.png" alt="Mail Icon"></li>
+          </ul>
+        </nav>
+      </div>
+
+      <div class="dashBorder"></div>
```

Comment:
Apply this border like I demonstrated in class last week. Instead of creating an entirely new `div` to add a border to you can add it to an element that already exists.